db/schema.rb

```diff
@@ -10,34 +12,33 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2022_05_20_200840) do
+ActiveRecord::Schema[7.0].define(version: 20_220_603_145_240) do
```

Comment:
I think the format that is set by default for the version number is more readable. I know rubocop changes it, but we can see if we can configure rubocop not to change it